From ea9bdabdf8b52c7457bf02dbf52846e31c71a6c7 Mon Sep 17 00:00:00 2001
From: Marek Belisko <marek.belisko@open-nandra.com>
Date: Wed, 5 Aug 2020 20:59:01 +0200
Subject: [PATCH] configs: sunxi: add Mender required options

Upstream-Status: Inappropriate
Signed-off-by: Mirza Krak <mirza.krak@northern.tech>
Signed-off-by: Marek Belisko <marek.belisko@open-nandra.com>

---
 include/configs/sunxi-common.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/configs/sunxi-common.h b/include/configs/sunxi-common.h
index b8ca77d031..5f355e64ff 100644
--- a/include/configs/sunxi-common.h
+++ b/include/configs/sunxi-common.h
@@ -99,6 +99,10 @@
 
 /* Ethernet support */
 
+#define CONFIG_BOOTCOUNT_LIMIT
+#define CONFIG_BOOTCOUNT_ENV
+#define CONFIG_SYS_REDUNDAND_ENVIRONMENT
+
 #ifdef CONFIG_ARM64
 /*
  * Boards seem to come with at least 512MB of DRAM.
